plugins/catalog-backend/api-report.md

```diff
@@ -368,7 +368,7 @@ export class DefaultCatalogCollatorFactory implements DocumentCollatorFactory {
   // (undocumented)
   getCollator(): Promise<Readable>;
   // (undocumented)
-  readonly type = 'software-catalog';
+  readonly type: string;
```

Comment:
`type` is a very generic name, would be it able to use something more descriptive like `documentType`

Reply:
I agree. I've changed the variable name to `documentType`. :)